extension-config/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "manifest_version": 2,
+  "name": "tota11y",
+  "description": "An accessibility visualization toolkit",
+  "version": "1.0",
+  "icons": {
+    "16": "assets/icon-16.png",
+    "24": "assets/icon-24.png",
+    "48": "assets/icon-48.png",
+    "32": "assets/icon-128.png"
+  },
+  "browser_action": {
+    "default_title": "tota11y",
+    "default_popup": "popup.html"
+  },
+  "content_scripts": [
+    {
+      "matches": [
+        "https://*/*",
+        "http://*/*"
```

Comment:
Even though it's 2019, some sites still use http, so it needs to be supported. YOLO